### ISSUE TITLE
clojure.lang.ExceptionInfo: No matching ctor found for class java.lang.IllegalArgumentException [at line 66, column 22]

### ISSUE THREAD

**Issue by dkain** [rymndhng/release-on-push-action#20]
have been running into the error `Generating release...
clojure.lang.ExceptionInfo: No matching ctor found for class java.lang.IllegalArgumentException [at line 66, column 22]` for the last couple of days; haven't updated our job(s) recently. The jobs look like: 

```
name: Name

on:
  push:
    branches:
      - release

jobs:
  release-on-push:
    runs-on: ubuntu-latest
    env:
      GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
    steps:
      - uses: rymndhng/release-on-push-action@master
        with:
          strategy: minor
```

has anyone run into something like this before?

**Comment by rymndhng** [comment]
Hey there! 

Sorry about that. I've been trying to rewrite the internals to make it easier to add features and must have overlooked something. 

What I would suggest is pinning the action to an older version, i.e. v0.9.3. I'll investigate more deeply into this. 

Example:

```
jobs:
  release-on-push:
    runs-on: ubuntu-latest
    env:
      GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
    steps:
      - uses: rymndhng/release-on-push-action@v0.9.3
        with:
          strategy: minor
```

The README should probably suggest pinning to a suggested version. Using `master` is a bit dangerous 😅.

**Comment by dkain** [comment]
haha was about to do that as a workaround, we were playing it a little fast and loose pinning to master 😅. Thanks so much for your quick reply!

**Comment by rymndhng** [comment]
Ah, I think I know the issue. So something I realized a couple of days ago was that the documentation and code were using different names and the default scenario was falling through in unexpected ways.

Moving forward, the property name should be renamed from `strategy` to `bump_scheme_version`.

The latest version on master includes a fix via #21.

**Comment by dkain** [comment]
thanks so much!
